File: Easy/Java/2710. Remove Trailing Zeros From a String.java
class Solution {
    public String removeTrailingZeros(String num) {
        int end = num.length();

        for(int i = num.length() - 1; 0 <= i; i--){
            if(num.charAt(i) != '0'){
                end = i + 1;
                break;
            }
        }

        return num.substring(0, end);
    }
}

File: Easy/Python/2710. Remove Trailing Zeros From a String.py
class Solution(object):
    def removeTrailingZeros(self, num):
        end = len(num)

        for i in range(len(num) - 1, -1, -1):
            if num[i] != '0':
                end = i + 1
                break

        return num[0:end]
